eclipse-che/che-operator
# Expose Che route on / path to avoid HostAlreadyClaimed error

### What does this PR do?
Expose Che route on / path to avoid HostAlreadyClaimed error.

### Screenshot/screencast of this PR
<details>

<summary>Before</summary>

![Screenshot_20210429_181014](https://user-images.githubusercontent.com/5887312/116695046-b5961f80-a9c8-11eb-805e-6e29050d8b0d.png)

</details>

<details>

<summary>After</summary>

![Screenshot_20210430_152906](https://user-images.githubusercontent.com/5887312/116695127-d1012a80-a9c8-11eb-93a7-7b650df1f08b.png)
</details>

### What issues does this PR fix or reference?
Fixes https://github.com/eclipse/che/issues/19721

### How to test this PR?

1. Deploy Eclipse Che
2. Check if both `che` and `dashboard` routes are accessible. 

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
